src/main/asciidoc/_chapters/protobuf.adoc

```diff
@@ -148,3 +148,67 @@ consider extending it also in
 Going forward, we will provide a new module of common types for use
 by CPEPs that will have the same guarantees against change as does our
 public API. TODO.
+
+=== protobuf changes for hbase-3.0.0 (HBASE-23797)
+Since hadoop also shades protobuf and bumps the version to 3.x, there is no
```

Comment:
... this in hadoop 3. what? Should you say that here? The hadoop version that moved pb?

Reply:
It's hadoop 3.3.x.

Reply:
k. Would be good to say in doc.